src/adagrams.js

```diff
@@ -1,15 +1,125 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const distOfLetters = {
```

Comment:
While we use the `const` keyword to ensure the values cannot be reassigned, we should also use capital letters to convey to other devs the value is fixed.
```suggestion
  const DIST_OF_LETTERS = {
```